packages/css-engine/src/core/style-sheet-atomic.test.ts

```diff
@@ -1,130 +1,176 @@
-import { describe, beforeEach, test, expect } from "@jest/globals";
```

Comment:
I am very confused about what's going on with file naming here, before this PR, there was style-sheet-atomic.ts and style-sheet-atomic.test.ts and the same thing for regular now tests for atomic are using regular style sheet?

very confusing everryhting you did with renaming